README.md

```diff
@@ -16,12 +16,14 @@
 
 ### Prerequisites
 
+* An OSX or Linux host. Building on Windows is only partially supported with
```

Comment:
Don't think we want to support the whole Makefile on Windows, but at least we can provide partial support for generating binaries.